extensions/amp-next-page/0.1/next-page-service.js

```diff
@@ -276,7 +273,7 @@ export class NextPageService {
    * one.
    * @param {number} nextPage Index of the next unseen page to use as the first
    *     recommendation in the list.
-   * @return {Element} Container element for the recommendations.
+   * @return {!Element} Container element for the recommendations.
```

Comment:
Let's break this files changes into a new PR.

Reply:
I've asked @aghassemi to review.